lms/services/blackboard_api/_basic.py

```diff
@@ -64,12 +64,6 @@ def __init__(
         self._oauth2_token_service = oauth2_token_service
 
     def get_token(self, authorization_code):
-        """
-        Get an access token from Blackboard and save it in the DB.
-
-        :raise services.HTTPError: if something goes wrong with the access
-            token request to Blackboard
-        """
```

Comment:
Moved the docstring onto the higher-level public method and didn't want it duplicated here